core/compilers.ts

```diff
@@ -33,8 +33,7 @@ export function compile(code: string, path: string): string {
   }
   if (path.endsWith(".ipynb")) {
     const notebookAsJson = stripNotebookOutputs(JSON.parse(code), path);
```

Comment:
I realise this is unrelated to this PR but: I think this is the wrong place for this to be happening! IMO output stripping should be happening in the `compile()` method of a NotebookAction class - following the same pattern for resolving/compiling SQLX/JS code into SQL - not here.

Reply:
Done.